content/tracks/bigdata.md

```diff
@@ -1,3 +1,5 @@
 ---
 title: Big Data
----
\ No newline at end of file
+---
+
+Big Data is leading and changing various industries and is inseparable from our lives. Big Data is also a very important part of ASF. ASF has so many big data projects, such as Hadoop, Hive, Spark, HBase, Kylin, Ozone, Cassandra, Doris, CarbonData, etc. In this topic, you will learn the cutting-edge trends of these technologies and the practical experience, principles, architecture analysis and other exciting content from first-line users
```

Comment:
We can add the website links of these projects.

Reply:
good idea, I have added the links, please review again, thx